engine/renderer/public/draw_stats.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <cstddef>
+
+class DrawStats
```

Comment:
Could be a part of a Command List abstraction to collect data implicitly on dispatch / draw calls